packaging/tests/scripts/output_logs_for_failures.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+TAS_DIRECTORY=$1
+
+cd ${TAS_DIRECTORY}
+
+failures=$(grep 'status="FAIL"' target/surefire-reports/testng-results.xml | sed 's|^.*[ ]name="\([^"]*\)".*$|\1|g')
```

Comment:
What does the `sed `part of this line do?

Reply:
The testng-results.xml file contains lines that look like:
```
<test-method signature="getRuleSetAndInheritedBy()[pri:0, instance:org.alfresco.rest.rules.GetRuleSetsTests@613a8ee1]" started-at="2022-09-16T09:48:12Z" name="getRuleSetAndInheritedBy" test-instance-name="" finished-at="2022-09-16T09:48:12Z" duration-ms="774" status="FAIL">
```

The sed command retrieves the `name="getRuleSetAndInheritedBy"` section and extracts the method name.

This is then matched in `TestSuite-output.txt` which contains lines like:
```
[main] 09:48:12 INFO  RestTest:77 - *** STARTING Test: [getRuleSetAndInheritedBy] ***
```